cadastrapp/cadastrapp.properties

```diff
@@ -0,0 +1,138 @@
+pdf.imageSize=200
+
+# Use for small plots, scale value in meter
+# For example for 1/2000 pdf.min.scale value will be 20
+pdf.min.scale=20
+# Those parameters are use enlarge PDF view when scale is higher than pdf.min.scale
+# There are percentage of view perimeter see #320 for more details
+pdf.ratio.mediumScale=0.2
+pdf.ratio.bigScale=0.1
+
+############################################################
+## In this section you can add as many differents basemaps you want
+## keep the same syntax for keys values pdf.baseMap.index.
+##
+## those basemaps will only be used to create image for pdf on server side
+## if several basemaps are set, the end user will be able to chose the one he wants
+## if only one basemap is set, end user won't have the choise
+## and if no baseMap URL are set then no basemap image will be added in pdf
+
+# BaseMap Title to be display in user interface
+pdf.baseMap.0.title=OSM Google Style - geobretagne
+pdf.baseMap.0.wms.url=http://osm.geobretagne.fr/service/wms?VERSION=1.1.1&Request=GetCapabilities&Service=WMS
+pdf.baseMap.0.layer.name=osm:google
+pdf.baseMap.0.format=image/png
+pdf.baseMap.0.SRS=EPSG:3857
+# only used when wms service need authentification
+# if empty no authentification is used
+pdf.baseMap.0.wms.username=
+pdf.baseMap.0.wms.password=
+
+# BaseMap Title to be display in user interface
+#pdf.baseMap.1.title=OSM Map Style - geobretagne
+#pdf.baseMap.1.wms.url=http://osm.geobretagne.fr/service/wms?VERSION=1.1.1&Request=GetCapabilities&Service=WMS
+#pdf.baseMap.1.layer.name=osm:map
+#pdf.baseMap.1.format=image/png
+#pdf.baseMap.1.SRS=EPSG:3857
+## only used when wms service need authentification
+## if empty no authentification is used
+#pdf.baseMap.1.wms.username=
+#pdf.baseMap.1.wms.password=
+###########################################################
+
+## information about WMS and WFS service
+# Here you can configure the layer name and with field contains the plot Id
+# depending if you are on Qgis or Arcopole model
+# This is use as well to generate image for BP
+# if this service does not support SLD_BODY WMS param please fill the addition parcelle.wms.url
+cadastre.wms.url=https://georchestra.mydomain.org/geoserver/pci/wms
+cadastre.wms.layer.name=pci:cadastre_2020
+# only used when wms service need authentification
+# if empty no authentification is used
+cadastre.wms.username=geoserver_privileged_user
+cadastre.wms.password=geoserver
+
+# Here you can configure the layer used to generate the plot selection on BP
+# let it empty if cadastre.wms.url support SLD_BODY WMS param
+# Note that it must support SLD_BODY WMS param
+parcelle.wms.url=
+parcelle.wms.layer.name=
+# The plot identifier field for the service
+parcelle.wms.layer.id=
+# only used when wms service need authentification
+# if empty no authentification is used
+parcelle.wms.username=
+parcelle.wms.password=
+
+cadastre.wfs.url=https://georchestra.mydomain.org/geoserver/wfs
+cadastre.wfs.layer.name=pci:parcelle_2020
```

Comment:
Does this mean we should also provide this layer by default with https://github.com/georchestra/geoserver_minimal_datadir ?

Reply:
especially since pci is available on https://cadastre.data.gouv.fr/data/dgfip-pci-vecteur/ ... but that means having a script to bootstrap the gs datadir fetching data from there, not going to import pci data in the repo :)